### PR TITLE
[DA-3532] Update Metrics Ingestion (AW2) workflow

### PR DESCRIPTION
**Pull request by dtharpeuno**
## Resolves *[ticket DA-3532]*
https://precisionmedicineinitiative.atlassian.net/browse/DA-3532

## Description of changes/additions
The current  AW2 ingestion process could make upwards for 2K+ db calls for 1K records in a manifest. This trims all the db calls to initially 2, but there could be subsequent others for odd conditions but this greatly trims the load on the workflow. Also started to move all short-read manifest ingestions to Abstract base classes (ABC), and a lot of other needed cleanup/refactors for the pipeline.

## Tests
- [] unit tests
** Updated a few tests that were wrong, but all tests should still pass since this was a refactor to mitigate memory issues on the app-engine instances

### CODE REVIEW

File: rdr_service/tools/tool_libs/genomic_utils.py [hunk 1]
```diff
@@ -1598,108 +1598,6 @@ def process_contamination_category_locally(self):
 
         return 0
 
-
-class IngestionClass(GenomicManifestBase):
```
Comment (brianmkimmel) [inline rdr_service/tools/tool_libs/genomic_utils.py:1602]: Why is IngestionClass being removed?

Comment (dtharpeuno) [reply rdr_service/tools/tool_libs/genomic_utils.py:1602]: I am trying to clean up tools that aren't in use anymore, do you still use this?

Comment (dtharpeuno) [reply rdr_service/tools/tool_libs/genomic_utils.py:1602]: Its fine I added back for now. d1732d751bdb885c127d67e6d32b3e983814f924

Comment (brianmkimmel) [reply rdr_service/tools/tool_libs/genomic_utils.py:1602]: Thanks, I use this tool when running AW1 reingestions.

**Review by j-kanuch — APPROVED** [review]
Nice

**Review by brianmkimmel — APPROVED** [review]

**Pull request merged**